docusaurus/docusaurus.config.js

```diff
@@ -464,11 +464,11 @@ const config = {
             position: 'right',
             activeBaseRegex: '/docs/downloads',
           },
-          // {
-          //   to: '/blog',
-          //   label: 'Blog',
-          //   position: 'right'
-          // },
+          {
+            to: 'https://blog.openziti.io/',
```

Comment:
doesn't render in vercel. dunno how it looks in dark mode but.... i suppose it's fine?

Reply:
It shows up when I peek at the Vercel preview and looks fine to me in dark mode.